src/tensorlake/functions_sdk/invocation_state/invocation_state.py

```diff
@@ -9,14 +9,24 @@ class InvocationState:
     Each new invocation starts with an empty state (empty set of key-value pairs).
     A value can be any CloudPickleSerializer serializable object."""
 
+    def __init__(self):
+        self.timers: Dict[str, float] = {}
+        self.counters: Dict[str, int] = {}
```

Comment:
Nit. These metrics are part of a single task run state (not whole graph invocations state).